src/payment/payment-strategy-registry.ts

```diff
@@ -41,6 +41,15 @@ export default class PaymentStrategyRegistry extends Registry<PaymentStrategy, P
     }
 
     private _getToken(paymentMethod: PaymentMethod): PaymentStrategyType {
+        const ppsdkFeatureOn =
+            this._store.getState()
+                .config.getStoreConfig()
+                ?.checkoutSettings.features['PAYMENTS-6806.enable_ppsdk_strategy'];
+
+        if (ppsdkFeatureOn && paymentMethod.type === PaymentStrategyType.PPSDK) {
```

Comment:
what happens if `paymentMethod.type === PaymentStrategyType.PPSDK` and `!ppsdkFeatureOn` ? Maybe throw an exception explaining that is not available?

Reply:
I think if `!ppsdkFeatureOn`, we want it to return `PaymentStrategyType.CREDIT_CARD` which currently happens.
Is that what we want @leeBigCommerce ?

Reply:
Would be good to add a unit test for that scenario then, to ensure the expected behaviour is tested and maintained.

Reply:
Yep, this is largely an extra precaution to gate off in progress code.

I've got a test to prove that it isn't selected in that scenario: https://github.com/bigcommerce/checkout-sdk-js/pull/1128/files#diff-7ab1b1fea01093913fc2ae689daa95fb1326449a6eaa1ca2d51d9fd13b50b354R76

It doesn't assert which strategy is chosen, as I didn't want to hardcode it to expect a specific default (which is tested elsewhere).